src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAICrafting.java

```diff
@@ -173,13 +175,13 @@ protected IAIState getRecipe()
             {
                 remaining = inputStorage.getAmount();
             }
-            else if (!ItemStackUtils.isEmpty(container) && ItemStackUtils.compareItemStacksIgnoreStackSize(inputStorage.getItemStack(), container , false, true))
+            else if (ItemStackUtils.isEmpty(container) || !ItemStackUtils.compareItemStacksIgnoreStackSize(inputStorage.getItemStack(), container , false, true))
```

Comment:
Again this change. These kind of suttle changes need to be documented if it is not a hotfix.
Why? This logic does not seem to make sense to me.

Reply:
reverted